### PR TITLE
Removes questionable language from perks.

### PR DESCRIPTION
**Pull request by Takykardi**
## About The Pull Request
Update: due to conversation on balance, non-QoL improvements struck from PR and edits made to focus on fixing the dogwhistle-y language. 

## Changelog
:cl:
fix: removed problematic language from marqua
/:cl:

### CODE REVIEW

**Review by Michiyamenotehifunana — APPROVED** [review]
Everything checks out here.

**Comment by Lynx22330** [comment]
Ooh. I didn't see this. 

I do want to preface that silicons are able to self heal. Someone mentioned it to me that the Will to Survive perk was possibly intended to be a sort of contender for silicons as they are able to self repair without any additional help (Of course, at least surface level damage)

This should be kept in mind, I feel.

**Comment by JoltzW** [comment]
I will add a additional that many races have very strong passives. Chit'mains ignore pain entirely, Akula can't be embedded and can get massive stat boosts with a perk, Naramads can drink water for infinite tricord. I do not think that humans are overtuned in this regard and if anything it will put them under par to many other races that already exist.

File: code/datums/perks/racial.dm [hunk 1]
```diff
@@ -59,7 +59,7 @@
 	cooldown_time = world.time + 25 MINUTES
 	user.visible_message("[user] suddenly looks lost in thought, their focus elsewhere for a moment.", "You clear your mind and feel your thoughts focusing into a single stream of brilliance.", "You hear the calming silence, as if someone nearby is thinking deeply.")
 	log_and_message_admins("used their [src] perk.")
-	user.reagents.add_reagent("marquatol", 10)
+	user.reagents.add_reagent("marquatol", 5)
```
Comment (Michiyamenotehifunana) [inline code/datums/perks/racial.dm:62]: Please revert these changes.

File: code/datums/perks/racial.dm [hunk 2]
```diff
@@ -103,10 +103,10 @@
 	if(world.time < cooldown_time)
 		to_chat(usr, SPAN_NOTICE("The human body can only take so much, you'll need more time before you've recovered enough to use this again."))
 		return FALSE
-	cooldown_time = world.time + 10 MINUTES
+	cooldown_time = world.time + 30 MINUTES
```
Comment (Michiyamenotehifunana) [inline code/datums/perks/racial.dm:106]: Ditto.

File: code/datums/perks/racial.dm [hunk 3]
```diff
@@ -103,10 +103,10 @@
...
 	user.visible_message("[user] grits their teeth and begins breathing slowly.", "You grit your teeth and remind yourself you ain't got time to bleed!")
 	log_and_message_admins("used their [src] perk.")
-	user.reagents.add_reagent("adrenol", 5)
+	user.reagents.add_reagent("adrenol", 3) //Enough to get you out of danger or towards medical.
```
Comment (Michiyamenotehifunana) [inline code/datums/perks/racial.dm:109]: Ditto.

File: code/modules/reagents/reagents/race.dm [hunk 4]
```diff
@@ -47,7 +47,7 @@
 	scannable = TRUE
 
 /datum/reagent/medicine/hadrenol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
-	M.heal_organ_damage(1, 0.8, 4, 2) // Barely buffed up Hustimdol without the sleepyness, any more would be too good. Remember this has a half hour cooldown.
+	M.heal_organ_damage(1, 0.8, 4, 2) // Barely buffed up Hustimdol without the sleepyness, any more would be too good. Equinox edit: cooldown returned to one half hour.
```
Comment (Michiyamenotehifunana) [inline code/modules/reagents/reagents/race.dm:50]: Ditto.

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
In light of recent discussions, we'll be requesting that non-QoL balance tweaks (e.g. tweaking numbers to affect the overall difficulty of the game rather than to remove excessive tedium) be held off for the time being. As such, please undo the reagent amount and cooldown time changes.

**Comment by Takykardi** [comment]
PR scope changed. Now it's only removing problematic language.

**Review by Michiyamenotehifunana — APPROVED** [review]
Looks good to me.
